src/components/Card/Card.css

```diff
@@ -76,6 +79,13 @@
   padding: 5px 10px;
   margin: 0 2.5px 5px;
   line-height: 1;
+
+  /** Prevent text selection */
+  user-select: none;
+  -moz-user-select: none;
```

Comment:
Don't need prefix. 
https://facebook.github.io/create-react-app/docs/post-processing-css

Reply:
Oh, right, been using JSS too often recently and thought CSS needs these overall 😅